pyccel/epyccel.py

```diff
@@ -169,22 +167,38 @@ def epyccel_seq(function_or_module, *,
     bcast : {True, False}
         If False, only root process loads accelerated function/module (default: True) (for parallel mode). 
     """
+    # Store current directory
+    base_dirpath = os.getcwd()
+
+    if isinstance(function_or_module, FunctionType):
+        dirpath = os.getcwd()
+
+    elif isinstance(function_or_module, ModuleType):
+        dirpath = os.path.dirname(function_or_module.__file__)
+
+    # Define working directory 'folder'
+    if folder is None:
+        folder = os.path.dirname(dirpath)
+    else:
+        folder = os.path.abspath(folder)
+
+    # Define directory name and path for epyccel files
+    epyccel_dirname = '__epyccel__'+os.environ.get('PYTEST_XDIST_WORKER','')
```

Comment:
Could you add some empty spaces around the `+` binary operator? And one after the comma?

Reply:
Fixed b245ece3